### PR TITLE
[Docs] Update `fileio.md`

### PR DESCRIPTION
**Pull request by Xiangxu-0103**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/advanced_tutorials/fileio.md [hunk 1]
```diff
@@ -37,21 +37,21 @@ with open('test.yaml', 'w') as f:
 ```python
 from mmengine import load, dump
 
-# 从 s3 文件读取数据
+# 从文件中读取数据
```
Comment (HAOCHENYE) [inline docs/zh_cn/advanced_tutorials/fileio.md:40]: s3 是一种文件存储，建议保留该字段

File: docs/zh_cn/advanced_tutorials/fileio.md [hunk 2]
```diff
@@ -37,21 +37,21 @@ with open('test.yaml', 'w') as f:
...
 data = load('s3://bucket-name/test.json')
 data = load('s3://bucket-name/test.yaml')
 data = load('s3://bucket-name/test.pkl')
 
-# 将数据保存至 s3 文件 (根据文件名后缀反推文件类型)
+# 将数据保存至文件 (根据文件名后缀反推文件类型)
```
Comment (HAOCHENYE) [inline docs/zh_cn/advanced_tutorials/fileio.md:45]: 同上

**Review by ice-tong — APPROVED** [review]
LGTM with 1 comments

**Review by C1rN09 — COMMENTED** [review]
同上意见

**Review by C1rN09 — APPROVED** [review]

**Review by zhouzaida — APPROVED** [review]

**Pull request merged**